UnitTests/meson.build

```diff
@@ -29,23 +31,30 @@ su2_cfd_tests_dd = files(['Common/simple_directdiff_test.cpp'])
 
 if get_option('enable-tests')
   if get_option('enable-normal')
+    unit_test_deps = [
+        fs.copyfile('Common/containers/lookuptable.drg', 'lookuptable.drg'),
+        fs.copyfile('Common/containers/lookuptable_3D.drg', 'lookuptable_3D.drg'),
+    ]
     unit_test_files = su2_cfd_tests + files(['test_driver.cpp'])
     test_driver = executable(
         'test_driver',
         unit_test_files,
-        install : true,
         dependencies : [su2_cfd_dep, common_dep, su2_deps, catch2_dep],
         cpp_args: ['-fPIC', default_warning_flags, su2_cpp_args]
     )
-    test('Catch2 test driver', test_driver)
+    test(
+        'Catch2 test driver',
+        test_driver,
+        depends: unit_test_deps,
+        workdir: meson.current_build_dir(),
+    )
   endif
 
   if get_option('enable-autodiff')
     unit_test_files_ad = su2_cfd_tests_ad + files(['test_driver.cpp'])
     test_driver_AD = executable(
         'test_driver_AD',
         unit_test_files_ad,
-        install : true,
```

Comment:
We would need to change the regression workflow to find the unit test binaries somewhere else. It's simpler if you add a meson option to skip installing them by default, and then we can install them in the regression test script.

Reply:
Thanks! I'll have a look!

Reply:
But... The documentation indicates the unit tests should be run from within the build dir (https://su2code.github.io/docs_v7/Running-Unit-Tests/). Shouldn't the regressions test script do the same, then?

(I'm not an SU2 pro... Maybe the regression test script tests for more and/or other things, which would invalidate the above.)

Reply:
Option 2 would require a change to the `compileSU2.sh` script. I doubt that that will be reflected anytime soon, which results in the tests failing until the container is updated.

@pcarruscag I think options 3 above is what you suggested, but would still either break the documented way to run unit tests (`ninja -C builddir test`), or require one to specify a workdir in `regression.yml`, which seems impossible.

What is the intention of tests? Should they be run before installing, to ensure one doesn't install a broken binary, or should they be run after installing?